app/components/Rewards.tsx

```diff
@@ -0,0 +1,136 @@
+import NftEventsRewards from "../components/nft_events_rewards/NftEventsRewards";
+
+const MarketPlace: {
+  title: string;
+  desc: string;
+  price: number;
+  currency: string;
+  buttonText: string;
+}[] = [
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+  {
+    title: "VIP Event Access",
+    desc: "Get VIP access to exclusive blockchain events",
+    price: 200,
+    currency: "wrld",
+    buttonText: "Redeem",
+  },
+];
+
+const nftDashboardData = {
+    events: [
+      {
+        id: "event1",
+        title: "Blockchain Conference",
+        date: "2024-01-01",
+        image: "/images/event1.png",
+      },
+    ],
+    rewards: [
+      {
+        type: "Airdrop",
+        eventName: "NFT Launch",
+        amount: 4000,
+        currency: "WRLD",
+        date: "2024-01-10",
+      },
+    ],
+    certifications: [
+      {
+        id: "cert1",
+        title: "Certified Blockchain Developer",
+        image: "/images/cert1.png",
+      },
+    ],
+  };
+  
+
+function Rewards() {
+  return (
+    <div className="flex flex-col gap-4 my-3 pr-6">
+      <p>Rewards</p>
+      <div className="bg-white flex flex-col gap-2 rounded-xl px-5 py-3 text-black font-semibold ">
+        <p>Token Balance</p>
+        <h2 className="text-4xl uppercase ">4,000 Wrld</h2>
+        <p className="text-xs font-light">≈ $125.00 USD</p>
+      </div>
+      {/* nft reward/ events */}
+      <NftEventsRewards
```

Comment:
This is not what is supposed to be imported here. the one you are supposed to import can be found in the reward-nft file